packages/makepot/README.md

```diff
@@ -0,0 +1,40 @@
+# `makepot` - **Github Action**
+
+This action creates POT file for WordPress plugin or theme.
+
+## Input
+
+| Name                | Description                                                                                                     |
+| ------------------- | --------------------------------------------------------------------------------------------------------------- |
+| `save-path`         | POT File path **(Required)**                                                                                    |
+| `slug`              | Plugin/Theme slug **(Required)**                                                                                |
+| `text-domain`       | Plugin/Theme text-domain (Default: `slug`)                                                                      |
+| `package-name`      | Plugin/Theme name to use in `Project-Id-Version` header                                                         |
+| `ignore-domain`     | Ignore the text domain completely and extract strings with any text domain.                                     |
+| `include`           | Comma-separated list of files and paths that should be used for string extraction.                              |
+| `exclude`           | Comma-separated list of files and paths that should be skipped for string extraction.                           |
+| `headers`           | Array in JSON format of custom headers which will be added to the POT file. Defaults to empty array.            |
+| `headers-json-file` | Path of the JSON file to read the POT headers from. Must be relative to root directory. Preferred over headers. |
```

Comment:
not sure how these args are inquired, but you might take a look at this package, maybe there is smth useful: https://github.com/SBoudrias/Inquirer.js

Reply:
They are to be added to workflow file.